OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -101,20 +101,9 @@ public ShroudSource(SourceType type, PPos[] projectedCells)
 		// Per-cell cache of the resolved cell type (shroud/fog/visible)
 		readonly ProjectedCellLayer<ShroudCellType> resolvedType;
 
+		bool oldDisabled;
```

Comment:
Perhaps this should use a different name. I find using `old` doesn't really fit here and just makes it confusing. Perhaps keep the custom `set` and add `disabledDirty = true` to it or a comment / summary / remark

Reply:
Changed to keep custom getter/setter as suggested, and promoted `disabledChanged` to a field.